app/models/user.rb

```diff
@@ -3,10 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :confirmable, :validatable
+
```

Comment:
- [ ] I can see you made some changes but as requested in the [previous review](https://github.com/Harrynoqs/blog-app/pull/9#discussion_r1161696072) It's recommended that you keep the second logic that includes the `: confirmable` module, as it allows for email confirmation of user accounts. However, you can remove the first duplicate line to avoid confusion and keep your code cleaner. As required by the previous reviewer, I can see you remove the second on instead of the first one :wink:  :smile: kindly fix this 
![Screenshot from 2023-04-10 16-23-56](https://user-images.githubusercontent.com/88894525/230923152-1f244bc9-d9ef-4579-b9dd-5705e2fb2160.png)